examples/tinywl/main.cpp

```diff
@@ -208,12 +218,23 @@ bool TinywlServer::start()
     return true;
 }
 
+
 void TinywlServer::onNewOutput(QWOutput *output)
 {
     Q_ASSERT(output);
     outputs.append(output);
-
     output->initRender(allocator, renderer);
+#if WLR_VERSION_MINOR > 17
+    auto *state = new wlr_output_state;
+    wlr_output_state_init(state);
+    if (!wl_list_empty(&output->handle()->modes)) {
+        auto *mode = output->preferredMode();
+        wlr_output_state_set_mode(state, mode);
+    }
+    wlr_output_state_set_enabled(state, true);
+    if (!output->commitState(state))
+        return;
+#else
```

Comment:
`QWOutput::commitState` can use with wlroots 0.17 too, so we can remove this brach.